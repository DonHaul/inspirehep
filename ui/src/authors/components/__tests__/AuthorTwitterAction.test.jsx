import React from 'react';
import { render } from '@testing-library/react';

import AuthorTwitterAction from '../AuthorTwitterAction';

describe('AuthorTwitterAction', () => {
  it('renders with twitter', () => {
    const { asFragment } = render(<AuthorTwitterAction twitter="harunurhan" />);
    expect(asFragment()).toMatchSnapshot();
  });
});
