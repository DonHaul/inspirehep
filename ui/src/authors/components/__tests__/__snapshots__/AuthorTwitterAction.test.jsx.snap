// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthorTwitterAction renders with twitter 1`] = `
<DocumentFragment>
  <span
    class="__UserAction__"
  >
    <a
      href="//x.com/harunurhan"
      target="_blank"
    >
      <span
        aria-label="x"
        class="anticon anticon-x"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="x"
          fill="currentColor"
          fill-rule="evenodd"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M921 912L601.11 445.75l.55.43L890.08 112H793.7L558.74 384 372.15 112H119.37l298.65 435.31-.04-.04L103 912h96.39L460.6 609.38 668.2 912zM333.96 184.73l448.83 654.54H706.4L257.2 184.73z"
          />
        </svg>
      </span>
    </a>
  </span>
</DocumentFragment>
`;
